<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
  </author>
 <class>WdgOptionsHeightMap</class>
 <widget class="QWidget" name="WdgOptionsHeightMap">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>319</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item column="0" row="0">
    <widget class="QLabel" name="fileSizeDescLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>File size:</string>
     </property>
    </widget>
   </item>
   <item column="1" row="0">
    <widget class="QLabel" name="fileSizeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>File size (bytes)</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item column="0" row="3">
    <widget class="QLabel" name="endiannessLabel">
     <property name="text">
      <string>Endianness:</string>
     </property>
    </widget>
   </item>
   <item column="1" row="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="radioLittle">
       <property name="toolTip">
        <string>Little Endian</string>
       </property>
       <property name="text">
        <string>&amp;Little</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">endiannessButtonGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBig">
       <property name="toolTip">
        <string>Big Endian</string>
       </property>
       <property name="text">
        <string>&amp;Big</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">endiannessButtonGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item colspan="2" column="0" row="5">
    <widget class="QGroupBox" name="dimensionsGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item colspan="3" column="0" row="4">
       <widget class="QPushButton" name="guessButton">
        <property name="toolTip">
         <string>Guess width and height based on file size and bits per pixel. Only values set to 0 will be updated.</string>
        </property>
        <property name="text">
         <string>Guess dimensions</string>
        </property>
       </widget>
      </item>
      <item column="0" row="3">
       <widget class="QLabel" name="heightLabel">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item column="2" row="1">
       <widget class="QLabel" name="pxLabel">
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
      <item column="1" row="3">
       <widget class="QSpinBox" name="heightInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Height of image in pixels.</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item column="2" row="3">
       <widget class="QLabel" name="pxLabel2">
        <property name="text">
         <string>px</string>
        </property>
       </widget>
      </item>
      <item column="1" row="1">
       <widget class="QSpinBox" name="widthInput">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Width of image in pixels.</string>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
       </widget>
      </item>
      <item column="0" row="1">
       <widget class="QLabel" name="widthLabel">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item colspan="2" column="0" row="6">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item column="0" row="1">
    <widget class="QLabel" name="bppDescLabel">
     <property name="text">
      <string>Bits per pixel:</string>
     </property>
    </widget>
   </item>
   <item column="1" row="1">
    <widget class="QLabel" name="bppLabel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item column="0" row="2">
    <widget class="QLabel" name="typeDescLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item column="1" row="2">
    <widget class="QLabel" name="typeLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioLittle</tabstop>
  <tabstop>radioBig</tabstop>
  <tabstop>widthInput</tabstop>
  <tabstop>heightInput</tabstop>
  <tabstop>guessButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="endiannessButtonGroup"/>
 </buttongroups>
</ui>